R/PACKAGES.R

```diff
@@ -165,7 +158,16 @@ update_packages <- function(old_local_packages, new_local_packages) {
 
   old_packages <- old[!old$Package %in% new$Package, ]
   new_packages <- rbind(old_packages, new)
-  new_packages[order(new_packages$Package), columns_ordered]
+
+  # NOTE: filtering needed in case the initial PACKAGES file was empty
+  new_packages <- new_packages[!is.na(new_packages$Package), ]
```

Comment:
Note that if there are no packages the dimensions would be dop and new_packages could become a vector

```suggestion
  new_packages <- new_packages[!is.na(new_packages$Package), , drop = FALSE]
```